sys/include/net/netopt.h

```diff
@@ -270,6 +270,17 @@ typedef enum {
      */
     NETOPT_IS_WIRED,
 
+    /**
+     * @brief Get the Phy link status.
```

Comment:
Can you adapt for the documentation style in #8655, please?

Reply:
Of course, I've adapted the `@brief` to the style of #8655 and reworded the full description a bit to specify behaviour more precise.